packages/at_sync_ui_flutter/CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## 1.0.10
+- **CHORE**: Bumped all dependency versions
+
```

Comment:
Last released version of sync package is `1.0.8`.
`1.0.10` changes can be  combined with `1.0.9`